createdisk-library.sh

```diff
@@ -95,7 +96,7 @@ function update_json_description {
         | ${JQ} '.clusterInfo.kubeConfig = "kubeconfig"' \
         | ${JQ} '.nodes[0].kind[0] = "master"' \
         | ${JQ} '.nodes[0].kind[1] = "worker"' \
-        | ${JQ} ".nodes[0].hostname = \"${VM_PREFIX}-master-0\"" \
+        | ${JQ} ".nodes[0].hostname = \"${vm_name}\"" \
```

Comment:
I'd use `${VM_NAME}` directly rather than passing this as an argument.

Reply:
Again not blocker, something can be take care as part of refactor.

Reply:
This makes the changes simpler, if you agree with the change, not much point in adding complexity now in order to remove it later on.